quodlibet/browsers/albums/main.py

```diff
@@ -424,7 +424,8 @@ class AlbumList(Browser, util.InstanceTracker, VisibleUpdate,
     __last_render = None
     __last_render_surface = None
 
-    _PATTERN_FN = os.path.join(quodlibet.get_user_dir(), "album_pattern")
+    # TODO: Maybe on the Config folder?
+    _PATTERN_FN = os.path.join(quodlibet.get_data_dir(), "album_pattern")
```

Comment:
Not sure this is data or configuration.

Reply:
Seems like config to me... though I'm not even sure I always understand XDG's distinction.

Reply:
Config is something the user explicitly changes (like a preference). Data is for non-volatile data (not cache). The boundaries are a bit fuzzy.

This is the last pattern the user typed, right? if so, it should be data.